chadjones839/Capstone
# Login through Logout

# Description
- Login sets session storage and pushes user to the Discovery page.
- Discovery page contains Navbar
- Navbar link to Profile, Discovery, and Chat.
- Profile page contains logout button which clears session storage, and redirects user to login page 
- Discovery currently only built for employer account type profiles. Discovery page maps through database of users and displays all employer cards.
- Profile currently displays static details.
- Register button displayed on home page, but not yet linking to a registration page.

## Type of change

# Testing Instructions
- Populate and serve json file in api folder on port 5002 using key/value pairs from example.json
- Serve src files on port 3000
- Open devtools and use mobile view with iphone X template.
- On login page, use one of the example users login credentials.
- Click login button.
- You should be redirected to /discovery
- Click the profile icon on the left side of navbar to open profile page.
- On profile page, click the logout icon in the upper left corner of the screen.
- This should redirect you back to the login page.

# Checklist:
